### PR TITLE
Create zh_cn.json

### PR DESCRIPTION
**Pull request by 200542053**

### CODE REVIEW

File: src/main/resources/assets/traverse/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+itemGroup.traverse=遍历
```
Comment (coderbot16) [inline src/main/resources/assets/traverse/lang/zh_cn.json]: This line looks like it would cause a JSON parse error, I assume it should be removed?

Comment (200542053) [reply src/main/resources/assets/traverse/lang/zh_cn.json]: Oh sure, What I meant for this was that "traverse" means "遍历" lol.

**Comment by coderbot16** [comment]
Looks good to me, thanks! This will be pushed out in the next update.

**Comment by Prospector** [comment]
Thank you for this!

**Pull request merged**